react/package.json

```diff
@@ -26,18 +26,13 @@
   "license": "ISC",
   "devDependencies": {
     "@types/node": "^20.3.1",
+    "esbuild": "^0.20.2",
     "playwright": "^1.35.1",
-    "typescript": "^5.1.3",
-    "rollup": "^3.28.1",
-    "@rollup/plugin-commonjs": "^25.0.4",
-    "@rollup/plugin-node-resolve": "^15.2.1",
-    "@rollup/plugin-terser": "^0.4.3"
+    "typescript": "^5.1.3"
   },
   "peerDependencies": {
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
-  },
-  "dependencies": {
+    "react-dom": "^18.2.0",
     "dsd-polyfill": "1.0.0"
```

Comment:
Did you want to move `dsd-polyfill` to `devDependencies` here?

Reply:
yes. It's only used when building in dev. The final production result is included with the package

Reply:
Sorry, I meant now it's listed as a `peerDependency` and should perhaps be moved to L28,5.

Reply:
Oh! Good catch!